docker-compose.yml

```diff
@@ -0,0 +1,41 @@
+version: '3.3'
+
+services:
+  db:
+    image: mysql:5.7
+    ports:
+      - "3307:3306"
+    environment:
+      MYSQL_ROOT_PASSWORD: "password"
+
+  local:
+    build: .
+    command: /app/local.sh
+    ports:
+      - 4000
+    environment: &ENV
+      DB_PASSWORD: password
+      DB_NAME: eventful_dev
+      DB_HOST: db
+      MIX_ENV: dev
+    volumes:
+        - ./:/app
+    links:
+      - db
+    depends_on:
+      - db
+
+  test:
```

Comment:
I dont think we need this in the docker-compose. This is supposed to help others start the app not run unit tests. if someone wants to run tests, they can run the command on the image.

Reply:
we're using this in our travis, it has different envs, it has the same image but with different flow

Reply:
aslo, won't it useful to run the tests right a way ?